src/code.cloudfoundry.org/policy-server/api/api.go

```diff
@@ -114,8 +116,3 @@ type Tag struct {
 	Tag  string `json:"tag"`
 	Type string `json:"type"`
 }
-
-type Space struct {
```

Comment:
Out of curiosity, why did we delete this and add `SpaceEntity` in cc_client/client.go?

Reply:
It was to resolve cross-package dependencies between cc_client + api after moving the fake into cc_client